README.md

```diff
@@ -124,7 +124,10 @@ More information is available in [ADR 12](docs/arch/adr-12.md) and [ADR 17](docs
 
 The yarn build scripts default to production for libraries if `PANOPTES_ENV` is not specified. The apps are always built to the production API.
 - `NODE_ENV`: the [webpack build mode](https://webpack.js.org/configuration/mode/) for libraries and the NextJS apps (production, development or undefined.)
-- `APP_ENV`: sets the [Sentry environment](https://docs.sentry.io/enriching-error-data/environments/?platform=javascript) (development, staging or production) when reporting errors.
+- `APP_ENV`: the deployment environment:
```

Comment:
Suggestion to still document setting the Sentry environment here. APP_ENV is used in `initializeSentry.js` in both app-project and app-content-pages.

Reply:
Sentry logs the deployment environment so that we can filter events depending on whether they came from `frontend.preview.zooniverse.org` or `www.zooniverse.org`. I think 'Sentry environment' was bad wording in the original text. I can add a note here but I think that might also be in the Sentry docs.

Reply:
I should probably add a link to this: https://docs.sentry.io/product/sentry-basics/environments/

Reply:
Done!